contributors/guide/reviewer-guidelines.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Code Reviewal Guidelines"
+weight: 1
+slug: "reviewer-guidelines" 
+---
+
+*This Section is new and in progress. Expect this document to change often*
+
+**Table Of Contents**
+
+- [Reviewer Guidelines](#reviewer-guidelines)
+    + [Reviewing Process](#reviewing-process)
+    + [Reviewing Ideology](#reviewing-ideology)
+    + [Something To Avoid](#something-to-avoid)
+    + [Additional Readings](#additional-readings)
+
+## Reviewing Process
+
+Typically, there are various ways to review a PR or patch. Based on how major is the PR or how much different section it affects in the project, You may choose your typical way for review.
+
+### Flakes/Test Fixes
+- If the PR is to fix any test in the repository's CI , you can easily review the change by seeing the `Green Ticks` in the PR Checks.
+
+### Typo's/Very Minor Change
+- If the PR is about a typo in the repository, any grammer suggestion or it comprises of a very minore change, You can see the differences in the code made by using the GitHub UI, and check if the PR satifies all the criterias.
+
+### A Patch/Minor Chage
+- If the PR is a patch or a minor change in the repository, You can browse through the changes line of codes through the GitHub UI. You can use the `Split Diff View` for easy view of each LOC and also you can see changes made in the each commit of the patch using the UI.
+
+### Major Change:
+- If the PR is a major change which may affect a lot of section of your repository, You can pull the local changes of the PR into your local system and browse through each and every change made in the PR. You can also test & play around with the changes done in the PR to ensure each and every change is as expected.
+- To fetch the Pull Request locally, Use this -
+```sh
+$ git fetch origin pull/ID/head:BRANCHNAME
```

Comment:
This is somewhat inaccurate for the [kubernetes GitHub workflow](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md). With a fork/PR based workflow you'll be pulling from upstream and you have a couple options.
```
git checkout -b '<pr-to-review>'
git pull upstream/<ID>/head
```
or
```
git remote add <PR author> <PR author's forked repo>
git fetch <PR author>
git checkout --track <PR Author>/<PR author's branch>
```
example:
```
# their PR is based off branch awesome-feature
git remote add herpderp git@github.com:herpderp/kubernetes.git
git fetch herpderp
git checkout --track herpderp/awesome-feature
```